### DISCUSSION TITLE
I need suggestions please

### DISCUSSION THREAD

**Discussion by codevault4all** [community/community#83142] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello, I am new to coding and I need suggestions as to which websites I can visit to start learning web development.

**Comment by anuragjain-git** [comment]
There are several great websites but few popular ones are:

1. [MDN Web Docs](https://developer.mozilla.org/en-US/docs/)
2. [W3Schools CSS Tutorial](https://www.w3schools.com/)
3. [Free Code Camp](https://www.freecodecamp.org/)

**Reply by codevault4all** [reply]
Thank you very much for this information

**Comment by codesmith-emmy** [comment, marked as answer]
If you would prefer to learn from video tutorials, these YouTube channels could be helpful:

1. [Travesty Media](https://m.youtube.com/@TraversyMedia)
2. [Web Dev Simplified](https://m.youtube.com/@WebDevSimplified) 
3. [The Net Ninja](https://m.youtube.com/@NetNinja)

These websites below are also great for learning web development:

1. [Free Code Camp](https://www.freecodecamp.org/)
2. [MDN Docs](https://developer.mozilla.org/en-US/docs/)
3. [Web3Schools](https://www.w3schools.com/)

**Reply by codevault4all** [reply]
Thanks

**Comment by PrathibhaSithu** [comment]
For a beginner I suggest
[W3schools](https://www.w3schools.com/)
[Khan Academy ](https://www.khanacademy.org/)
[Net Ninja](https://www.youtube.com/@NetNinja)

**Reply by anuragjain-git** [reply]
All of them are good, But i would prefer W3schools or MDN docs.

**Comment by rajiv2004-cloud** [comment]
Welcome to the world of coding! 

If you are intrested in learning video's
here are some channel's:
1.Traversy Media
2.The Net Ninja
3.Network chuck

few websites :
1.W3Schools
2.freeCodeCamp
3.GitHub(recoment this one a lot) - Explore projects on [GitHub](https://github.com/) to see real-world examples of code.

**Comment by codevault4all** [comment]
Thank you all for your contributions. I’m really  grateful.
